omnigibson/object_states/particle_modifier.py

```diff
@@ -323,7 +328,7 @@ def check_overlap():
             # Define the function for checking whether points are within the adjacency mesh
             def check_in_adjacency_mesh(particle_positions):
                 # Define the AABB bounds
-                lower, upper = self.obj.states[AABB].get_value() if self.link is None else self.link.aabb
+                lower, upper = self.link.aabb
```

Comment:
hmmm when `self.link is None`, shouldn't we still use the object bounding box instead of base link bounding box?

Reply:
keep as is, per our discussion